drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -764,8 +764,21 @@ void nrf_802154_tx_ack_started(const uint8_t *data)
 				data[FRAME_PENDING_BYTE] & FRAME_PENDING_BIT;
 }
 
-void nrf_802154_transmitted_timestamp_raw(const uint8_t *frame, uint8_t *ack,
-				int8_t power, uint8_t lqi, uint32_t ack_time)
+#if defined(CONFIG_NRF_802154_SER_HOST)
```

Comment:
I'm wondering now, shall we assert if both `CONFIG_NRF_802154_SER_HOST` and `CONFIG_NET_PKT_TIMESTAMP` are enabled (static assert)? With currnent proposal, the `nrf5_data.ack_frame.time` will not be set correctly, yet it will still be processed by the upper layer (OpenThread).

Reply:
If we do that it will break Zigbee build in NCS. They are using `CONFIG_NET_PKT_TIMESTAMP` together with `CONFIG_NRF_802154_SER_HOST`.

Reply:
Ok, I see now that the limitation only applies to the ACK frames. Nevermind then.